lib/user.rb

```diff
@@ -0,0 +1,22 @@
+require 'pg'
+
+class User
+  attr_reader :name, :password, :email
+
+  def initialize(name:, password:, email:)
+    @name = name
+    @password = password
+    @email = email
+  end
+
+  def self.add(name:, password:, email:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
```

Comment:
the class methods are getting really packed. Do you know a way to extract some of the code to another class?